docker/Makefile

```
@@ -197,7 +198,10 @@ clean-image-%:
 	-$(docker) rmi $(patsubst clean-image-%,katzenpost-%,$@)
 	-rm -fv $(patsubst clean-image-%,%,$@).stamp
 
-clean-local: stop
+clean-bin: stop
+	rm -vf $(net_name)/*.$(distro)
```

```suggestion
	rm -vf ./$(net_name)/*.$(distro)
```
how about this, then at works we nuke `.//.`? :-)

---

(which should be fine since it's not `rm -r`)